elements/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -121,11 +124,21 @@ export class TransactionPool {
 			return false;
 		}
 		// Check for minimum entrance fee to the TxPool and if its low then reject the incoming tx
-		if (incomingTx.fee < this._minimumEntranceFee) {
+		incomingTx.feePriority = this._calculateFeePriority(incomingTx);
+		if (incomingTx.feePriority < this._minEntranceFeePriority) {
 			return false;
 		}
+
 		// Check if incoming transaction fee is greater than the minimum fee in the TxPool if the TxPool is full
-		// TODO: Use feePriorityQueue to reject transaction with lowest fee when txPool is full
+		const lowestFeePriorityTrx = this._feePriorityQueue.peek();
+		if (
+			Object.keys(this._allTransactions).length === this._maxTransactions &&
```

Comment:
probably safer to check with `>`

Reply:
I'm checking `>=` as we are checking this condition before adding so it should be checking for equal as well